### PR TITLE
fix docker dev deployment

### PR DESCRIPTION
**Pull request by olamilekan000**
```
change fixes docker deployment for development
```
Screenshot 2024-05-03 at 19 02 00

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -21,26 +21,26 @@ services:
...
+    #       context: ./
+    #       dockerfile: Dockerfile.dev
+    #     entrypoint: ["./cmd", "scheduler", "--config", "convoy.json"]
+    #     volumes:
+    #       - ./configs/local/convoy.json:/convoy.json
+    #     restart: on-failure
+    #     depends_on:
+    #         - postgres
+    #         - redis_server
+    #         - typesense
```
Comment (jirevwe) [inline docker-compose.dev.yml:35]: We can remove this completely

File: web/ui/dashboard/package.json [hunk 2]
```diff
@@ -12,48 +12,47 @@
...
-		"@types/react": "^18.0.21",
-		"axios": "^1.6.7",
-		"date-fns": "^2.27.0",
-		"monaco-editor": "^0.34.1",
-		"posthog-js": "^1.36.0",
-		"prismjs": "^1.29.0",
-		"rxjs": "^6.6.0",
-		"tslib": "^2.3.1",
-		"zone.js": "^0.11.4"
+    "@angular/common": "^14.0.3",
```
Comment (jirevwe) [inline web/ui/dashboard/package.json:15]: @Oluwadaminiola can you check the changes in this file?

Comment (olamilekan000) [reply web/ui/dashboard/package.json:15]: This is not clear

Comment (olamilekan000) [reply web/ui/dashboard/package.json:15]: I did update some of the dependencies version

Comment (jirevwe) [reply web/ui/dashboard/package.json:15]: Yeah, I know; I'm not sure what they do, so I want her to review them.

Comment (Oluwadaminiola) [reply web/ui/dashboard/package.json:15]: there's no breaking change here @jirevwe

Comment (jirevwe) [reply web/ui/dashboard/package.json:15]: Thanks. LGTM @olamilekan000

**Pull request closed without merging**